cms/djangoapps/contentstore/views/tests/test_certificates.py

```diff
@@ -298,6 +299,22 @@ def test_certificate_info_in_response(self):
         self.assertEqual(data[0]['description'], 'Test description')
         self.assertEqual(data[0]['version'], CERTIFICATE_SCHEMA_VERSION)
 
+    @mock.patch.dict('django.conf.settings.FEATURES', {'CERTIFICATES_HTML_VIEW': True})
```

Comment:
@mattdrayer I did a little clean up in certificates.html that resulted in me having to fix the unit test.  I added an additional unit test as well.  Can you verify that these changes make sense and aren't covering up a bug?  I tested this through Studio UI and all looks good and bokchoy tests are passing.  Thanks.